tests/data/Block.1.sample1.json

```diff
@@ -1,927 +1,930 @@
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMN22O4APCNQ4ICXRCY2BWTPBHQDVFSS27CRERRYL6ZCRQBUV2BEFME"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMO22SYVHF5CXPIABI3F2MOL3L6XQF37GLU2W3SHEOIQZHXJSLFFS7W"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMP225FPLLGIRNIUJFP2QHAI3V6YCA3DT2HMBNQTMIO5EZPDH3EDIIA"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMQ22YRJOQQS3UGGE43SDTS6I7F35DF442IVDS4RSUF4MJOPF3DNQDO"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMR22F4O4GNVYTBNGBKLAK3ET25SHURBIVLHVTRDGMY5VVI3LPV4CUF"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMS22YAXJOT545SOSIN5ASLSAXKIDUHN7YMSRUAB7NB6QNCYLTQKQSK"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMT22RY3GWK5VP2N7QF7ULTQ2AS5OHIZQV5HKL5PFBIRGUC5MT27CUE"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMU22SCMIXRGW4Y7NK7BOJHB4ES3VFWLK7KFXC5Z3HSHNFFXGLEWD6I"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMV22ZPCUQWB7SLQNXRMW4XZ3RUUD6P5WMLACTZ6G7KOMSB3ZMCUMGN"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMW22LVW4KDMDFMCN4GN4FEFFSPMJTJ7H7PRUFQBNRKA4W4EHXCCEFU"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMX22AL3QGJT55FNSNAQDWBK3JGO5G3QYBHFCWZGI6CTHFWJCOYIPZF"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMY22XQ7UECVQBWTY7SIDTEFK4CGPCKLRQPCWS47XS3NPXKVPYVR7GX"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDMZ2253Q62MV5CWQ6EIVDZYFNI3BGWFIOHCUDNW5SK643JFNIA7T7O3"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDNA22QDAJRNNPD7BEJG4INCI2HOBUOVNQJLRBMNXHEDBXH6JQCQPWSW"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDNB22W7MUI36RFJFGBK6VDYQ23XOMPJYWE53LB6UZH4AYV6RUN76RC4"
-                },
-                {
-                    "value": "24400000000000",
-                    "address": "GDNC22KXQGDKH6EO2UIOWNCYOPMO6XZX7T6P4TH6XZ4UYSR6JOBGY5U6"
-                }
-            ],
-            "payload": ""
-        }
+    "header":{
```

Comment:
Please print out JSON Pretty and make it easier to compare with previous data.
https://jsonformatter.org/json-pretty-print